src/object/srv_obj_migrate.c

```diff
@@ -1808,9 +1813,14 @@ ds_migrate_query_status(uuid_t pool_uuid, uint32_t ver,
 	if (rc)
 		D_GOTO(out, rc);
 
+	/**
+	 * The object ULT is generated by 0 xstream, and dss_collective does not
+	 * do collective on 0 xstream
+	 **/
+	arg.obj_generated_ult += tls->mpt_obj_generated_ult;
```

Comment:
since you += mpt_obj_generated_ult here, seems should reset mpt_obj_generated_ult as zero then?

Reply:
oh, mpt_obj_generated_ult should not be reset, we will compare total number of generated with the executed number to know if the migration is done.

Reply:
ok, i see, every time zero it for migrate_check_one.